realm/realm-annotations-processor/src/main/java/io/realm/processor/RealmProxyClassGenerator.java

```diff
@@ -36,6 +36,7 @@
 
 
 public class RealmProxyClassGenerator {
+    private static final String OPTION_DEBUG = "debug";
```

Comment:
I think we should have more Realm specific name for annotation processor option name since it is seen by all annotation processors.

How about adding `realm.` prefix?